.rubocop.yml

```diff
@@ -28,6 +28,9 @@ Style/StringLiterals:
   EnforcedStyle: single_quotes
 Lint/ShadowingOuterLocalVariable:
   Enabled: false
+Metrics/BlockLength:
+  Exclude:
+  - 'spec/helpers/pension_chat_banner_helper_spec.rb'
```

Comment:
Shall we exclude the block length from the spec folder itself instead one file?

Reply:
@tomas-stefano yeah sure that can be done